shims/hdp26/hive/pom.xml

```diff
@@ -192,14 +192,13 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Bundle-Version>${pentaho-osgi-bundles.version}</Bundle-Version>
             <DynamicImport-Package>*</DynamicImport-Package>
             <Import-Package>
               org.pentaho.hadoop.shim.api.*,
               org.pentaho.hadoop.shim.spi.*,
             </Import-Package>
-            <_exportcontents>
-              org.apache.hadoop.hive.*;version=${bundle.export.version}
-            </_exportcontents>
+            <_exportcontents>org.apache.hadoop.hive;version=${bundle.export.version}</_exportcontents>
```

Comment:
Has hive been tested with this change? We left the wildcard off and it's fine to do as long as all the hive packages are covered by this.

Reply:
You are correct. I double checked and this should be a wildcard too.